src/backend/TrafficCourts/Citizen.Service/Models/Tickets/ViolationTicket.cs

```diff
@@ -0,0 +1,258 @@
+    [JsonPropertyName("issued_at_or_near_city")]
+    [MaxLength(100)]
+    public string? IssuedAtOrNearCityOrTown { get; set; }
+    #endregion
+
+    #region Offsence Details
+
+    /// <summary>
+    /// The violation ticket was issued for offence under the Motor Vehicle Act (MVA).
+    /// </summary>
+    [JsonPropertyName("is_mva_offence")]
+    public bool? IsMvaOffence { get; set; }
+
+    /// <summary>
+    /// The violation ticket was issued for offence under the Wildlife Act (WLA).
+    /// </summary>
+    [JsonPropertyName("is_wla_offence")]
+    public bool? IsWlaOffence { get; set; }
+
+    /// <summary>
+    /// The violation ticket was issued for offence under the Liquor Control and Licencing Act (LCA).
+    /// </summary>
+    [JsonPropertyName("is_lca_offence")]
+    public bool? IsLcaOffence { get; set; }
+
+    /// <summary>
+    /// The violation ticket was issued for offence under the Motor Carrier Act (MCA).
+    /// </summary>
+    [JsonPropertyName("is_mca_offence")]
+    public bool? IsMcaOffence { get; set; }
+
+    /// <summary>
+    /// The violation ticket was issued for offence under the Firearm Act (FAA).
+    /// </summary>
+    [JsonPropertyName("is_faa_offence")]
+    public bool? IsFaaOffence { get; set; }
+
+    /// <summary>
+    /// The violation ticket was issued for offence under the Transit Conduct and Safety Regs (TCR).
+    /// </summary>
+    [JsonPropertyName("is_tcr_offence")]
+    public bool? IsTcrOffence { get; set; }
+
+    /// <summary>
+    /// The violation ticket was issued for offence under the Commercial Transport Act (CTA).
+    /// </summary>
+    [JsonPropertyName("is_cta_offence")]
+    public bool? IsCtaOffence { get; set; }
+
+    /// <summary>
+    /// The violation ticket was issued for other.
+    /// </summary>
+    [JsonPropertyName("is_other_offence")]
+    public bool? IsOtherOffence { get; set; }
+
+    /// <summary>
+    /// The violation ticket was issued for other.
+    /// </summary>
+    [JsonPropertyName("other_offence_description")]
+    [MaxLength(100)]
+    public string? OtherOffenceDescription { get; set; }
+
+    /// <summary>
+    /// Represents the counts identified. Must have at least one and at most three counts.
+    /// </summary>
+    [JsonPropertyName("counts")]
+    [Required]
+    [Range(1, 3)]
+    public List<ViolationTicketCount> Counts { get; set; } = new List<ViolationTicketCount>();
+    #endregion
+
+    #region
+    /// <summary>
+    /// The designated provincial court hearing location. For example, Richmond, BC.
+    /// </summary>
+    [JsonPropertyName("provincial_court_hearing_location")]
+    [MaxLength(100)]
+    public string? ProvincialCourtHearingLocation { get; set; }
+
+    /// <summary>
+    /// The organization or detatchment location. For example, Delta Police.
+    /// </summary>
+    [JsonPropertyName("organization_location")]
+    [MaxLength(100)]
+    public string? OrganizationLocation { get; set; }
+    #endregion
+}
```

Comment:
In a way I might think it's better to have these model objects defined in the oracle-data-interface project instead of here. Then have both the staff-api and citizen-api use OpenApi to automatically pull the swagger spec to generate the model.  Otherwise the model is defined twice in two different places (here and in oracle) ... could lead to disparities between the definitions.